lua/packer/plugin_types/git.lua

```diff
@@ -108,9 +109,13 @@ git.setup = function(plugin)
   end
 
   if plugin.branch or plugin.tag then
-    install_cmd = fmt('%s --branch %s', install_cmd, plugin.branch and plugin.branch or plugin.tag)
+    table.insert(install_cmd, '--branch')
```

Comment:
Tiny nitpick: I'm gradually moving `packer` to the `tbl[#tbl + 1] = item` style of appending, since I've since been told it's the more "idiomatic"/slightly more performant (though not in a significant way for our use) approach.